tests/example.test.ts

```diff
@@ -0,0 +1,6 @@
+import { sum } from "@/utils/math";
+
+test('adds 1 + 2 to equal 3', () => {
+  expect(sum(1, 2)).toBe(3);
+});
+
```

Comment:
How is this example test different from math.test?

Reply:
I was trying to run multiple tests but I will remain with one test.